configman/tests/test_val_for_configobj.py

```diff
@@ -300,7 +303,7 @@ def test_write_ini_with_reference_value_froms(
 [c]
 
     # husband from Flintstones
-    #fred=stupid, deadly
+    #fred="stupid, deadly"
```

Comment:
that's  problem with configobj.  if you give it a string that has a comma, it will make the assumption that it should return a list object instead of a string.  We didn't have this problem when configman quoted everything in sight.  Now that configman is sparing in the use of quotes, it became a problem.